src/nwb_datajoint/spikesorting/spikesorting_curation.py

```diff
@@ -655,24 +713,26 @@ def make(self, key):
         # get the labels for the accepted units
         labels = {}
         for unit_id in accepted_units:
-            labels[unit_id] = ','.join(unit_labels[unit_id])
+            if unit_id in unit_labels:
+                labels[unit_id] = ','.join(unit_labels[unit_id])
 
         # Limit the metrics to accepted units
-        metrics = metrics.loc[accepted_units]
+        tmp_metrics = copy.deepcopy(metrics)
+        for metric in metrics:
+            for unit_id in metrics[metric]:
+                if unit_id not in accepted_units:
+                    del tmp_metrics[metric][unit_id]
+        metrics = tmp_metrics
 
         print(f'Found {len(accepted_units)} accepted units')
 
-        # exit out if there are no labels or no accepted units
-        if len(unit_labels) == 0 or len(accepted_units) == 0:
-            print(f'{key}: no accepted units found')
-            return
-
         # get the sorting and save it in the NWB file
         sorting = Curation.get_curated_sorting_extractor(key)
         recording = Curation.get_recording_extractor(key)
 
         # get the original units from the Automatic curation NWB file to get the sort interval information
-        orig_units = (SpikeSorting & key).fetch_nwb()[0]['units']
+        ss_key = (SpikeSorting & key).fetch1("KEY")
+        orig_units = (SpikeSorting & ss_key).fetch_nwb()[0]['units']
```

Comment:
I'm a little confused here. `SpikeSorting` does not save an analysis NWB file so I think this won't find anything. Is that right?